### PR TITLE
[test/plugin] immigrate test activemq scenario.

### PR DESCRIPTION
**Pull request by aderm**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues
#3583 
___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: test/plugin/scenarios/activemq-scenario/config/expectedData.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  instances:
+    - {activemq-scenario: nq 0}
+  operationNames:
+    - activemq-scenario: [ActiveMQ/Queue/test/Producer, ActiveMQ/Queue/test/Consumer, /activemq-scenario/case/activemq]
+  heartbeat: []
+segmentItems:
+- applicationCode: activemq-scenario
+  segmentSize: ge 2
+  segments:
+  - segmentId: /activemq-scenario/case/activemq
```
Comment (dmsolr) [inline test/plugin/scenarios/activemq-scenario/config/expectedData.yaml]: The `segmentId` is generated by SkyWalking.

File: test/plugin/scenarios/activemq-scenario/config/expectedData.yaml [hunk 2]
```diff
@@ -25,24 +25,8 @@ segmentItems:
 - applicationCode: activemq-scenario
   segmentSize: ge 2
   segments:
-  - segmentId: /activemq-scenario/case/activemq
+  - segmentId: not null
     spans:
-      - operationName: ActiveMQ/Queue/test/Producer
```
Comment (wu-sheng) [inline test/plugin/scenarios/activemq-scenario/config/expectedData.yaml]: Why remove this? We should have this in the expected data.

File: test/plugin/scenarios/activemq-scenario/config/expectedData.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        isError: false
+        spanType: Entry
+        peer: ''
+        peerId: 0
+        tags:
+        - {key: url, value: 'http://localhost:8080/activemq-scenario/case/activemq'}
+        - {key: http.method, value: GET}
+  - segmentId: not null
+    spans:
+      - operationName: ActiveMQ/Queue/test/Consumer
```
Comment (wu-sheng) [inline test/plugin/scenarios/activemq-scenario/config/expectedData.yaml]: No producer span, I am not sure why you remove it.

Comment (aderm) [reply test/plugin/scenarios/activemq-scenario/config/expectedData.yaml]: producer is exit span，i am not sure

Comment (wu-sheng) [reply test/plugin/scenarios/activemq-scenario/config/expectedData.yaml]: If no exit span, you don't have the ref in this span.

**Comment by wu-sheng** [comment]
Why remove ref? I am confused about your changes

File: test/plugin/scenarios/activemq-scenario/config/expectedData.yaml (hunk 3 above)
Comment (aderm) [reply test/plugin/scenarios/activemq-scenario/config/expectedData.yaml]: > If no exit span, you don't have the ref in this span.

This doesn't understand

**Pull request merged**